core/domain/param_domain.py

```diff
@@ -57,25 +87,49 @@ class ParamChange(object):
     """Value object for a parameter change."""
 
     def __init__(self, name, generator_id, customization_args):
-
+        """Initialze a ParamChange object with the specified arguments
+
+        Args:
+            name: unicode. The name of the parameter.
+            generator_id: unicode. The type of generator used to create the
+            parameter. eg. Copier
+            customization_args: dict. A dict containing all the arguments
+            including the value of the parameter.
```

Comment:
This is vague. Please be more specific about the possible keys that the dict may contain, as well as the meanings of those keys and the types of the corresponding values.